install-script/install.sh

```diff
@@ -1180,6 +1193,13 @@ if has_prepackaging_installation; then
    HAD_PREPACKAGING_INSTALLATION="true"
 fi
 
+if is_package_installed; then
+    use 'install.sh --upgrade' to upgrade otelcol-sumo
```

Comment:
You're missing an `echo` or something here?

Reply:
Oops, fixing now.

Reply:
Fixed!